src/interfaces/coral_web/src/hooks/conversation.tsx

```diff
@@ -56,10 +56,16 @@ export const useConversation = ({
     enabled: !!conversationId && !disabledOnMount,
     queryFn: async () => {
       try {
-        return await client.getConversation({
+        const conversation = await client.getConversation({
           conversationId: conversationId ?? '',
           signal: abortControllerRef.current?.signal,
         });
+        return {
+          ...conversation,
+          messages: conversation.messages.sort(
+            (a, b) => Date.parse(a.created_at) - Date.parse(b.created_at)
+          ),
+        };
```

Comment:
ideally the backend returns us this properly sorted

Reply:
This should fix the BE: https://github.com/cohere-ai/cohere-toolkit/pull/333
Would you mind double checking if it's working?

Reply:
looks like it so far, thanks! will remove this code :)